api/v1alpha1/internalrequest_types.go

```diff
@@ -60,8 +60,8 @@ type InternalRequestStatus struct {
 
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
-// +kubebuilder:printcolumn:name="Succeeded",type=string,JSONPath=`.status.conditions[?(@.type=="SucceededReason")].status`
-// +kubebuilder:printcolumn:name="Reason",type=string,JSONPath=`.status.conditions[?(@.type=="SucceededReason")].reason`
+// +kubebuilder:printcolumn:name="Succeeded",type=string,JSONPath=`.status.conditions[?(@.type=="Succeeded")].status`
```

Comment:
can you explain what this is doing?

Reply:
Showing a column (`Succeeded`) with the value of that condition status field. Conditions changed in a previous PR.